src/components/BusinessHoursSelect.vue

```diff
@@ -1,8 +1,19 @@
 <template>
-  <select :name="optionName" @change="inputEventHandler" v-model="selected">
-    <option v-show="isFirstRow(index) && onlyOneRow(hours)" value>{{
-      defaultText
-    }}</option>
+  <select :name="optionName"
+          :class="selectClass" 
+          v-model="selected" 
+          @focus="hidePlaceholder" 
+          @change="inputEventHandler" 
+          @blur="showPlaceholder">
+    <option v-show="isFirstRow(index) && onlyOneRow(hours)" 
```

Comment:
please re-order these properties according to this [guideline](https://aloware.atlassian.net/wiki/spaces/AM/pages/863961145/Guidelines#Talk2-%2F-Vue-(%5BinlineCard%5D-))